src/pages/home/report/ReportFooter.tsx

```diff
@@ -141,7 +142,7 @@ function ReportFooter({
                         />
                     )}
                     {isArchivedRoom && <ArchivedReportFooter report={report} />}
-                    {!canWriteInReport && <SystemChatReportFooterMessage />}
+                    {!isAnonymousUser && !canWriteInReport && isSystemChat && <SystemChatReportFooterMessage />}
```

Comment:
I believe the `!isAnonymousUser` was not necessary here anymore, but good enough for the CP